include/openPMD/Mesh.hpp

```diff
@@ -182,6 +182,14 @@ class Mesh : public BaseRecord< MeshRecordComponent >
 
     void flush_impl(std::string const&) override;
     void read() override;
+
+    /**
```

Comment:
Writing a doxygen string without an element that captures it will propagate the block to the next location where it can "dock". This will lead to undesirable results.
```suggestion
    /*
```